docs/examples/huggingface/playground.yaml

```diff
@@ -5,4 +5,21 @@ metadata:
 spec:
   replicas: 1
   modelClaim:
-    modelName: opt-125m
\ No newline at end of file
+    modelName: opt-125m
+  backendConfig:
+    name: vllm # Current support includes VLLM and SGLANG. More backends will be added in the future.
+    version: v0.5.1
+    args: # additional vllm args
+    - --load-format=auto
+    envs: # optional vllm envs
+    - name: VLLM_USE_MODELSCOPE
+      value: "false"
+    resources:
+      requests:
+        nvidia.com/gpu: 1
+        memory: 8Gi
+        cpu: 4
+      limits:
+        nvidia.com/gpu: 1
```

Comment:
Generally we should not add gpu resource here because the backend didn't need GPU, but the model needs, this is already set in the model config.

Reply:
https://github.com/InftyAI/llmaz/blob/c651c9f8bdcfe5a8410dda8c0768304a14f5ac7d/pkg/controller/inference/service_controller.go#L155 
`Service Controller` will extract a model `InferenceFlavor` info and generate responding `lws` object while `Playground Controller` won't. Maybe I missed something.

Reply:
see definition here: https://github.com/InftyAI/llmaz/blob/c651c9f8bdcfe5a8410dda8c0768304a14f5ac7d/api/inference/v1alpha1/config_types.go#L44-L48

Generally we'll handle resource requirements at two places, the first place is playground controller, if we set the backendConfig, https://github.com/InftyAI/llmaz/blob/c651c9f8bdcfe5a8410dda8c0768304a14f5ac7d/pkg/controller/inference/playground_controller.go#L201-L209. This is just incase we want to change the default cpu/mem set for backend, e.g. vllm has a default 4C8Gi set, with backendConfig, we can change this dynamically. But for GPU sets, as you pointed out, we'll extract the information from the Model obj in service controller.

So the right place to set the GPUs is at the Model obj, which we'll support fungibility in the future, like serving the llama2-70b, we may have configurations as:

```
inferenceFlavors:
  - name: a100-80gb
    requests:
        nvidia.com/gpu: 2
  - name: a100-40gb
     requests:
        nvidia.com/gpu: 4
```

Reply:
I get it. That will be a great feature. 
XD